instrumentation/apolloconfig-apolloclient-1.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.ctrip.framework.apollo")
+    module.set("apollo-client")
+    versions.set("[1.0.0,)")
+    assertInverse.set(true)
+  }
+}
+
+dependencies {
+  compileOnly("com.google.auto.value:auto-value-annotations")
+  annotationProcessor("com.google.auto.value:auto-value")
+
+  library("com.ctrip.framework.apollo:apollo-client:1.0.0")
+
+  testImplementation("com.ctrip.framework.apollo:apollo-client:1.1.0")
```

Comment:
You could use `testCompileOnly("com.ctrip.framework.apollo:apollo-client:1.1.0")` that way tests would run with `1.0.0`. Then you will have to make `TestConfigRepository` and `TestRepositoryChangeListener` private to avoid junit looking for test methods there.